modules/end-user-guide/pages/git-credential-store.adoc

```diff
@@ -1,25 +1,94 @@
 
-* The labels of the secret that is about to be created must match the set of labels configured in `che.workspace.provision.secret.labels` property of {prod-short}. The default labels are:
++
+[subs="+quotes,+attributes,+macros"]
+----
+https://__<username>__:__<token>__@__<git_server_hostname>__
+----
 
-* `app.kubernetes.io/part-of: che.eclipse.org` 
-* `app.kubernetes.io/component: workspace-secret`:
 +
-include::example$snip_{project-context}-git-credential-file-as-kubernetes-secret-into-the-file.adoc[leveloffset=+1]
+.A line in a `.git-credentials` file
+====
+`https://trailblazer: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@github.com`
+====
+
+. Select credentials from your `.git-credentials` file for the Secret. Encode the selected credentials to Base64 for the next step.
++
+[TIP]
+====
+CLI users can use the `base64` command as needed:
+
+* To encode all lines in the file:
+** `$ cat .git-credentials | base64 | tr -d '\n'`
+
+* To encode a selected line:
+** `$ echo -n '__<copied_and_pasted_line_from_.git-credentials>__' | base64`
+====
+
+. Create a new {platforms-name} secret in the user's {orch-namespace}.
++
+[source,yaml,subs="+quotes,+attributes,+macros"]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: git-credentials-secret
+  labels:
+    controller.devfile.io/git-credential: 'true' <1>
+    controller.devfile.io/watch-secret: 'true'
+  annotations:
+    controller.devfile.io/mount-path: /etc/secret <2>
+data:
+  credentials: __<Base64_content_of_.git-credentials>__ <3>
+----
++
+<1> The `controller.devfile.io/git-credential` label marks the Secret as containing Git credentials.
+<2> A custom absolute path in the {devworkspace} containers. The Secret is mounted as the `credentials` file at this path. The default path is `/`.
+<3> The selected content from `.git-credentials` that you encoded to Base64 in the previous step.
++
+[TIP]
+====
+You can create and apply multiple Git credentials Secrets in your user {orch-namespace}. All of them will be copied into one Secret that will be mounted to the {devworkspace} containers. For example, if you set the mount path to `/etc/secret`, then the one Secret with all of your Git credentials will be mounted at `/etc/secret/credentials`. You must set all Git credentials Secrets in your user {orch-namespace} to the same mount path. You can set the mount path to an arbitrary path because the mount path will be automatically set in the Git configuration file configured at `/etc/gitconfig`.
+====
+
+. Use the CLI or GUI to apply the Secret to the {orch-name} cluster of your organization's {prod-short} instance.
++
+For CLI users: Apply the Secret by entering it with `{orch-cli}`:
++
+[subs="+quotes,+attributes,+macros"]
+----
+$ {orch-cli} apply -f - <<EOF
+__<Secret_prepared_in_the_previous_step>__
+EOF
+----
+
+.Additional resources
 
+* link:https://git-scm.com/docs/git-credential-store[Documentation for git-credential-store]
```

Comment:
FYI, I don't know if we should be providing this link as an additional resource. The procedure in this PR does not involve the Git commands described on the linked page, so this makes me wonder if readers would find it confusing.

Reply:
Sounds good, removed